src/hooks/useInput/index.js

```diff
@@ -6,7 +6,7 @@ function useInput(initialValue) {
   return {
     value,
     setValue,
-    reset: () => setValue(""),
+    resetValue: () => setValue(""),
```

Comment:
Since we are resetting the value, I think it would be better to reset it to the `initialValue`. What do you think?

Reply:
Yes, I missed that should be the initialValue instead of "".